app/ts/devtools.ts

```diff
@@ -1,17 +1,17 @@
-/** 
+/**
  * Create an Angular panel from the Component Tree View
  * on Chrome Development Tools window.
 */
 chrome.devtools.panels.create(
-  'Angular', 
-  'app/images/angular.png', 
-  'app/js/batarangle/frontend/batarangle.html', 
+  'Angular2',
```

Comment:
There is `Batarangle` in the main panel. I think we should leave this one as Angular/Angular2. After all that's the framework we are debugging in this panel.